client/src/components/exercise/lab7/pages/ExerciseStart.js

```diff
@@ -0,0 +1,153 @@
+                  <li>
+                    For each round, a <b>threat level</b> will be detected in
+                    the system
+                  </li>
+                  <li>
+                    The system will restrict file access based on the{" "}
+                    <b>threat level</b> and the{" "}
+                    <b>file&lsquo;s sensitivity level</b>:
+                  </li>
+                  <div className={"tw-py-3"}>
+                    <table className={"tw-border-solid"}>
+                      <thead className={"tw-border-solid"}>
+                        <tr>
+                          <th className={"tw-px-16"}>Threat Level</th>
+                          <th className={"tw-px-16"}>Restricted Files</th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        <tr>
+                          <td className={"tw-text-center"}>High</td>
+                          <td className={"tw-text-center"}>
+                            Sensitivity Levels 4 and 5
+                          </td>
+                        </tr>
+                        <tr>
+                          <td className={"tw-text-center"}>Medium</td>
+                          <td className={"tw-text-center"}>
+                            Sensitivity Levels 2 and 3
+                          </td>
+                        </tr>
+                        <tr>
+                          <td className={"tw-text-center"}>Low</td>
+                          <td className={"tw-text-center"}>
+                            Sensitivity Level 1
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </div>
+                  <li>
+                    The simulation will notify you of whether the autonomous
+                    system made the correct restriction for each file by
+                    displaying <b>&quot;AI Correct&quot;</b> or{" "}
+                    <b>&quot;AI Incorrect&quot;</b>.
+                  </li>
+                  <li>
+                    The autonomous system&apos;s mistakes can be categorized
+                    into two types:
+                  </li>
+                  <ul className={"tw-py-1.5"}>
+                    <li>
+                      An <b>intrusion</b> occurs when a file is accessed when it
+                      should have been restricted.
+                    </li>
+                    <li>
+                      An <b>incorrect decision</b>, or{" "}
+                      <b>false positive (FP)</b>, occurs when a file is
+                      restricted when it should have been accessible.
+                    </li>
+                  </ul>
+                </ul>
+              </li>
+              <li>
+                At the end, a summary report containing all of the information
+                displayed throughout the simulation will be displayed
```

Comment:
full stop at the end?

Reply:
I'm a bit confused here, what do you mean?

Reply:
"." a period at the end I believe
@notcruz